### PR TITLE
Replace ``print`` statements with proper ``logging``

### PR DESCRIPTION
**Pull request by lkluft**
This PR:
* Introduces the usage of the ``logging`` standard module
* Adds a top-level convenience function ``typhon.set_loglevel()`` to control the logging level
* Replaces (almost) all calls of the ``print()`` function with the appropriate ``logging`` function
* Add information for developers on how to create log messages

In general, Typhon should not print any messages to ``stdout`` by default. To restore output, you can set the logging level by using the standard module
```python
logging.basicConfig(logging.INFO)
```
or the Typhon convenience function
```python
typhon.set_loglevel(logging.INFO)
```

Cheers,
Lukas

---
In addition:
* remove an outdated backup file 
* fix a bug in the ``DopplerWind`` class (switch between 'red' and 'blue' shift did not work)

### CODE REVIEW

**Comment by JohnMrziglod** [comment]
Good initiative! I will have a look into this next week!

**Comment by lkluft** [comment]
Okay, thanks! I tried to minimise the changes to your code and mainly used your existing logging mechanisms. It's up to you, if you want to refactor it even more 😉

File: typhon/retrieval/spareice/common.py [hunk 1]
```diff
@@ -134,8 +138,8 @@ def __init__(self, file=None, collocator=None, processes=10, verbose=2,
             processes: Number of processes to parallelize the training or
                 collocation search. 10 is the default. Best value depends on
                 your machine.
-            verbose: An integer value. The higher the value, the more debug
-                messages are printed.
+            verbose (int): Control ``GridSearchCV`` verbosity. The higher the
```
Comment (lkluft) [inline typhon/retrieval/spareice/common.py:141]: Here, one could implicitly use the logging level to determine what to pass to ``GridSearchCV``.
```python
level = logger.getEffectiveLevel()
if level >= logging.DEBUG:
    verbose = 2
```
or something like that. I was not sure, if that is needed or not. Therefore, I kept it as close to the existing code as possible.

**Review by olemke — APPROVED** [review]
Good initiative. In the future we could consider adding one or two convenience function for setting the log format, e.g. including a timestamp, the function and/or colored output. Maybe some fancy formatting such as:

```python
def set_default_logging_format(level=None, include_timestamp=True,
                               include_function=True):
    """Generate decently looking logging format string."""

    if level is None:
        level = logging.INFO

    color = "\033[1;%dm"
    reset = "\033[0m"
    black, red, green, yellow, blue, magenta, cyan, white = [
        color % (30 + i) for i in range(8)]
    logformat = '['
    if include_timestamp:
        logformat += f'{red}%(asctime)s.%(msecs)03d{reset}:'
    logformat += (f'{yellow}%(filename)s{reset}'
                  f':{blue}%(lineno)s{reset}')
    if include_function:
        logformat += f':{green}%(funcName)s{reset}'
    logformat += f'] %(message)s'

    logging.basicConfig(
        format=logformat,
        level=level,
        datefmt='%H:%M:%S')
```

**Review by olemke — APPROVED** [review]

**Pull request merged**